tendrl/ceph_integration/manager/pool_request_factory.py

```diff
@@ -21,7 +21,7 @@
 class PoolRequestFactory(RequestFactory):
 
     def _resolve_pool(self, pool_id):
-        osd_map = self._cluster_monitor.get_sync_object(OsdMap)
```

Comment:
BTW this is removed line. Anyway would check and remove any references of `_cluster_monitor` from source.